deploy/charts/rook-ceph/values.yaml

```diff
@@ -514,6 +514,18 @@ csi:
     # - topology.kubernetes.io/zone
     # - topology.rook.io/rack
 
+  readAffinity:
+    # -- Enable read affinity for rbd volumes.
+    enabled: false
+    # -- crushLocationLabels define which node labels to use
+    # as CRUSH location. This corresponds to the values used in
+    # ceph CRUSH map.
+    # @default -- labels listed [here](../CRDs/Cluster/ceph-cluster-crd.md#osd-topology)
+    crushLocationLabels:
```

Comment:
Do we need the crushLocationLabels field at all? Seems like the csi driver could just automatically detect which topology labels are on the node, then add them to the krbd options automatically. Or why does the user need to specify it?

Reply:
> Do we need the crushLocationLabels field at all? Seems like the csi driver could just automatically detect which topology labels are on the node, then add them to the krbd options automatically. Or why does the user need to specify it?

CephCSI inherently does not have knowledge about topology labels, 
It needs to be told which labels it needs to look at to extract crush location. 

By default Rook feeds it the default set of labels which itself uses for OSD.

Edge case where user may need to user this option according to me is explained here.
https://github.com/rook/rook/pull/11631#discussion_r1099634744

Reply:
This will be a very edge case. Makes sense to keep it, but how about the following to keep the default simple?
They should only need to add additional custom labels. By default rook would automatically tell CSI about all the supported OSD topology labels and the hostname label.